stylelint-polaris/tests/index.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+const stylelint = require('stylelint');
+
+const fixturePath = path.join(process.cwd(), 'tests/fixture.css');
+const fileContent = fs.readFileSync(fixturePath, 'utf-8');
+
+(async () => {
+  const {results} = await stylelint.lint({
+    config: {
+      extends: '@shopify/stylelint-polaris',
```

Comment:
```suggestion
      extends: '../index.js',
```
Does this need to be a relative path to the `stylelint-polaris/index.js` file? Not immediately understanding how `config.extends` is able to resolve this bare module specifier (`@shopify/stylelint-polaris`).

Reply:
Good point here... I think `@shopify/stylelint-polaris` would resolve to the symlinked resources in the `node_modules` folder.

So it should be pointing to the `main` file defined in the `package.json`. In this case, either should be fine.

Reply:
So just by nature of `stylelint-polaris` being a yarn workspace the package is symlinked in the root `node_modules` 🤯 ? For some reason I thought it had to be referenced in the `"dependencies":{}` of another workspace's `package.json`, TIL!

Reply:
I think it makes sense to do the `../` for the test though. Good learnings anyway :)